specification/implementation/metadata/general-requirements/index.md

```diff
@@ -1,66 +1,65 @@
 ## 5.1. General requirements for metadata in a CSIP Information Package
-The number one consideration when discussing metadata requirements is, as with the rest of this specification, the need for interoperability. In more detail, the focus is on high-level technical interoperability and tasks which allow an Information Package to be prepared, transferred and received regardless of the institutions and tools involved. These tasks include:
+The primary consideration when considering metadata requirements is, as with the rest of this specification, that of interoperability. That is to say that we focus on high-level technical interoperability and tasks which allow an Information Package to be prepared, transferred and received regardless of the institutions and tools involved. These tasks include:
 
 - Identifying uniquely an Information Package and its components;
 - Validating an Information Package;
 - Validating the contents of an Information Package;
-- Proving the authenticity of the Information Package;
+- Establishing the authenticity of the Information Package;
```

Comment:
I am not a native English speaker, so disregard if my understanding is not correct, but I understand the word "establish" to be more "active", and "moderating", and "creating something new" and even though we as archives are and must be active in order to do digital preservation, then I read this sentence as if we create the authenticity. I understand our role and tasks more as we establish the organization around digital objects in order to make sure their authenticity will remain intact.

Reply:
I agree that the archives should absolutely not "create authenticity" but preserve the authenticity that we are given. And for that, we need to find out exactly how much authenticity there is in the IP, so we need to test, analyse, validate, determine, examine. "Establish" is a better word for that than "prove," as prove implies convincing someone else, while here the goal is to find out the truth for ourselves.

The bullets here are a list of "tasks that allow an IP to be prepared, transferred and received," which is very technical, so I think "validate the authenticity of the IP" is the most accurate, although it repeats "validate."

Reply:
Ok, so I was going for an "establishing the facts" style use, am tempted to stick with the wording. Would like to avoid overloading validation.